Shuffle job paths to avoid cache-vs-live collisions: Part II [BA-6236] (#5429)
broadinstitute/cromwell: CHANGELOG.md

SELECT table_rows FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_SCHEMA = 'cromwell' AND table_name = 'JOB_STORE_SIMPLETON_ENTRY';
```

#### Execution Directory Layout (cache copies)

> Attempts to copy call cached results will no longer occupy the same directory as the first attempt to run the task, should the cache copy fail. 

Reviewer: [ToL]
```suggestion
When an attempt to copy call cache results fails, its result will no longer occupy the same directory as the first attempt to run the task.
```

Author: My only nit with that rewording is that it's true even if the attempt to call cache does not fail.

Reviewer: Ah, well if my version is straight up wrong then it's more than a nit

Author: I think the original commits the same sin. I'll try to make this more correct

SELECT table_rows FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_SCHEMA = 'cromwell' AND table_name = 'JOB_STORE_SIMPLETON_ENTRY';
```

#### Execution Directory Layout (cache copies)

Attempts to copy call cached results will no longer occupy the same directory as the first attempt to run the task, should the cache copy fail. 

> Instead they will get their own subdirectory of the call root, eg:

Reviewer: [ToL]
```suggestion
Instead they will get their own `cacheCopy` subdirectory of the call root, eg:
```

```

> [...]/callRoot/

Reviewer: [ToL]
Might a "before" be helpful in addition to the "after"? This was tricky to figure out and I say it doesn't hurt to write down what's in our heads.
